### ISSUE TITLE
Typescript types?

### ISSUE THREAD

**Issue by 324Luke** [ndelvalle/v-click-outside#210]
This would allow for usage within Typescript projects.

**Comment by ndelvalle** [comment]
Yeah, that would be cool, @324Luke are you familiar with TS to provide some help?

**Comment by 324Luke** [comment]
@ndelvalle I work with TypeScript fairly often. I have never worked with typescript declaration files though.

**Comment by astagi** [comment]
@lukewhrit @ndelvalle I had the same problem on a Vue project written in TS, I tried this fix https://github.com/ndelvalle/v-click-outside/pull/265 . Let me know if it works for you :)

**Comment by josuemartinezz** [comment]
> @lukewhrit @ndelvalle I had the same problem on a Vue project written in TS, I tried this fix #265 . Let me know if it works for you :)

Works ⚡️

**Comment by josuemartinezz** [comment]
Add the following line in the configuration file `tsconfig.json`

``` ts
    "types": [
      ...
      "types/v-click-outside.d.ts"
    ]
```

In your root folder create a directory `types/v-click-outside.d.ts`

``` ts
declare module 'v-click-outside';
```

That's all 😜
